src/Tezos/BinaryCoding.cpp

```diff
@@ -19,7 +19,7 @@ std::string base58ToHex(const std::string& string, size_t prefixLength, uint8_t*
     if (decoded.size() < prefixLength) {
         return "";
     }
-    return "00" + TW::hex(decoded.data() + prefixLength, decoded.data() + decoded.size());
+    return TW::hex(decoded.data() + prefixLength, decoded.data() + decoded.size());
```

Comment:
You are correct to remove this because any arbitrary base58 should not be prepended with "00". This worked because this function is only called by `forgePublicKey`. 

To correct:
- Leave this line as is in this PR.
- Change Forging.cpp:58 to be:
```
    auto decoded = "00" + base58ToHex(pk, 4, prefix.data());

```